spec/forms/new_phone_form_spec.rb

```diff
@@ -151,6 +151,22 @@
       expect(result.errors).to be_empty
     end
 
+    context 'when the user has already added the number' do
+
+      it 'is invalid' do
+        phone = PhoneFormatter.format(params[:phone], country_code: 'US')
+        phone_configuration = create(:phone_configuration, user: user, phone: phone)
+
+        result = subject.submit(params)
+
+        expect(result).to be_kind_of(FormResponse)
+        expect(result.success?).to eq(false)
+        expect(result.errors[:phone]).to eq([I18n.t('errors.messages.phone_duplicate')])
+      end
+
+
+    end
```

Comment:
Rubocop will probably whine, but we should probably remove these blank linez

```suggestion
      end
    end
```